src/CoreApi/IFilestoreApi.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Ipfs.CoreApi
+{
+    /// <summary>
+    /// Abstraction for the file store api.
+    /// </summary>
+    public interface IFilestoreApi
```

Comment:
`<param/>`s in this interface should have documentation copied from the RPC API, linked in the original ticket.

![image](https://github.com/user-attachments/assets/48aa9437-f519-4bec-9812-cac57e493fad)